pkg/block/errors.go

```diff
@@ -5,4 +5,7 @@ import "github.com/pkg/errors"
 var (
 	ErrDataNotFound          = errors.New("not found")
 	ErrOperationNotSupported = errors.New("operation not supported")
+	ErrAsyncCopyInfo         = errors.New("failed to get copy status")
+	ErrAsyncCopyFailed       = errors.New("async copy failed")
+	ErrAsyncCopyAborted      = errors.New("async copy unexpected abort")
```

Comment:
In other adapters we don't have async copy. We need this error because we are not getting an error from the adapter on the copy status - we need to generate it ourselves. Note that when we call StartCopyFromURL, if we get an error we return the error itself and not wrapping it using our own error. 
I hope I answered the right question here :)

Reply:
Understood - can we have ErrAsyncCopyFailed error - that will be used for ErrAsyncCopyAborted and ErrAsyncCopyInvalidStatus?

Reply:
Aborted is an invalid status since we don't expect aborted status to show - same as failed